charts/prometheus-customizations/templates/influx-secret.yml

```diff
@@ -0,0 +1,12 @@
+{{- if .Values.influxdb.enabled }}
+apiVersion: 'kubernetes-client.io/v1'
+kind: ExternalSecret
+metadata:
+  name: influx-secret
+spec:
+  backendType: secretsManager
+  region: us-west-2
+  data:
+    - key: "{{ .Values.influxdb.secret_key }}"
```

Comment:
so this presumes - when a helm release is attempted - that someone has already setup 
1. access for the k8s resources to connect to AWS secrets (sure that's part of deploying secretsManager, will need to find where that is to dig deeper for my own understanding)
2. someone has created those secrets in AWS (we would... terraform this somehow? manually create them?)

Reply:
for 1, the aws access is part of the terraform external secrets setup https://github.com/mozilla-it/terraform-modules/blob/master/aws/eks/external_secrets.tf

for 2, we've been manually creating them, or if an automated system can randomly generate it, using tf to populate on occasion.  In this case, since the secret is from influxdb, created by hand, this one will probably be manually created.

Reply:
got it, thanks!